services/director-v2/src/simcore_service_director_v2/modules/comp_scheduler/_dask_scheduler.py

```diff
@@ -121,11 +136,15 @@ async def _start_tasks(
                     client.send_computation_tasks(
                         user_id=user_id,
                         project_id=project_id,
-                        cluster_id=pipeline_params.cluster_id,
+                        cluster_id=(
+                            comp_run.cluster_id
+                            if comp_run.cluster_id is not None
+                            else DEFAULT_CLUSTER_ID
+                        ),
                         tasks={node_id: task.image},
                         hardware_info=task.hardware_info,
-                        callback=pipeline_params.wake_up,
-                        metadata=pipeline_params.run_metadata,
+                        callback=wake_up_callback,
```

Comment:
remind me. what was this wake up callback for?

Reply:
this triggers the scheduling faster than 5 seconds